SomyaSinghal/Day2/03.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+int year(int x)
+{
+    if (x % 4 == 0)
```

Comment:
Correct the logic, input 1900. It will tell 1900 as leap year but in reality, it is not.